homeassistant/components/device_tracker/tomato.py

```diff
@@ -14,14 +14,20 @@
 import homeassistant.helpers.config_validation as cv
 from homeassistant.components.device_tracker import (
     DOMAIN, PLATFORM_SCHEMA, DeviceScanner)
-from homeassistant.const import CONF_HOST, CONF_PASSWORD, CONF_USERNAME
+from homeassistant.const import (
+    CONF_HOST, CONF_PORT, CONF_SSL, CONF_VERIFY_SSL,
+    CONF_PASSWORD, CONF_USERNAME)
 
 CONF_HTTP_ID = 'http_id'
 
-_LOGGER = logging.getLogger(__name__)
+_LOGGER = logging.getLogger("{}.{}".format(__name__, "Tomato"))
```

Comment:
No need to change that.

Reply:
Okay. I was just pulling the logger on L51 to match here. I can revert the change and still pull L51 out.